ihmc-interfaces/src/main/messages/ihmc_interfaces/ihmc_common_msgs/msg/ConfirmableRequestMessage.msg

```diff
@@ -7,5 +7,5 @@
 uint32[<=10] request_numbers
 
 # Recent confirmation numbers, the values of received request numbers
-# The size of this is roughly the max amount of expected dropped messages in 1 second
-uint32[<=10] confirmation_numbers
+# The size of this must be greater than or equal to the max timeout
+uint32[<=250] confirmation_numbers
```

Comment:
Made the confirmation numbers be able to hold far beyond any of our CRDT sync frequencies, since I guess if you freeze every tick, then you could get 30 in 1 second, so possible some confirmations would not come back for the full duration. I don't think this actually would cause an issue, but this is more consistent and safer in wild scenarios